src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -2175,5 +2180,13 @@ public void setSandboxType(SandboxType sandboxType) {
             sandboxProgramName = Constants.PC2_INTERNAL_SANDBOX_PROGRAM_NAME;
         }
     }
+    
+    public boolean isOnLoadDataFilesLoadSamplesFirst() {
+        return onLoadDataFilesLoadSamplesFirst;
+    }
+    
+    public void setOnLoadDataFilesLoadSamplesFirst(boolean onLoadDataFilesLoadSamplesFirst) {
```

Comment:
Same comment as above regarding the method name.

Reply:
pushed fix.